SYNC_UPSTREAM.md

```diff
@@ -205,58 +202,69 @@ bin/hcc `bin/hcc-config --build --cxxflags --ldflags` \
 ./a.out ; echo $?
 ```
 
-### Push HCC Clang submodule
+### Push and Create Pull Request for HCC Clang submodule
 
 - change to HCC directory
 - `cd clang`
-- `git checkout clang_tot_upgrade`
-- `git push`
+- `git push clang_fork merge_20170626:merge_20170626`
+Create a pull request to merge `merge_20170626` from your clang fork
+into https://github.com/RadeonOpenCompute/hcc-clang-upgrade
+on branch `clang_tot_upgrade`.
 
-Following steps are to ensure "master" branch are kept the same
-as "clang_tot_upgrade" branch.
-- `git checkout master`
-- `git merge clang_tot_upgrade --no-edit`
-- `git push`
-
-Finally switch back to "clang_tot_upgrade" branch.
-- `git checkout clang_tot_upgrade`
-
-### Push amd-hcc LLD submodule
+### Push and Create Pull Request for amd-hcc LLD submodule
 
 - change to HCC directory
 - `cd lld`
-- `git checkout amd-hcc`
-- `git push`
+- `git push lld_fork merge_20170626:merge_20170626`
+Create a pull request to merge `merge_20170626` from your lld fork
+into https://github.com/RadeonOpenCompute/lld.git
+on branch `amd-hcc`.
 
-### Push amd-hcc LLVM submodule
+### Push and Create Pull Request for amd-hcc LLVM submodule
 
 - change to HCC directory
 - `cd compiler`
-- `git checkout amd-hcc`
-- `git push`
+- `git push llvm_fork merge_20170626:merge_20170626`
+Create a pull request to merge `merge_20170626` from your llvm fork
+into https://github.com/RadeonOpenCompute/llvm.git
+on branch `amd-hcc`.
 
-### Push amd-hcc COMPILER-RT submodule
+### Push and Create Pull Request for amd-hcc COMPILER-RT submodule
 
 - change to HCC directory
 - `cd compiler-rt`
-- `git checkout amd-hcc`
-- `git push`
+- `git push compiler-rt_fork merge_20170626:merge_20170626`
+Create a pull request to merge `merge_20170626` from your compiler-rt fork
+into https://github.com/RadeonOpenCompute/compiler-rt
+on branch `amd-hcc`.
 
-### Push remove-promote-change-addr-space ROCm-Device-Libs submodule
 
-- change to HCC directory
-- `cd rocdl`
-- `git checkout remove-promote-change-addr-space`
-- `git push`
+*** Wait until all Pull Requests are approved and merged. ***
```

Comment:
@aaronenyeshi since now the whole process can't be completed without the PRs be merged. perhaps you may want to add another paragraph or 2 to illustrate the interaction between one who initates the PRs, and the one who reviews them?